functions/src/config/constants.js

```diff
@@ -27,10 +27,26 @@ const TIME_PERIODS = {
   "22": "10pm-12am"
 };
 
+const TIME_PERIODS_US = {
+  "8": "08:00-10:00",
+  "10": "10:00-12:00",
+  "12": "12:00-14:00",
+  "14": "14:00-16:00",
+  "16": "16:00-18:00",
+  "18": "18:00-20:00",
+  "20": "20:00-22:00",
+  "22": "22:00-00:00",
+  "0": "00:00-02:00",
+  "2": "02:00-04:00",
+  "4": "04:00-06:00",
+  "6": "06:00-08:00"
```

Comment:
the time periods will need to match what is in the front end
let me get them now

Reply:
Spot on you had that already 👏